### PR TITLE
Request review for submission

### PR DESCRIPTION
**Pull request by MegumiKim**
Please review my social app, for CA. 
It seems there are many different ways to customise Bootstrap components and I tried to implement all to see how it works.

I am afraid my HTML has become too messy/long. Is it preferred to change variables over making custom components scss files (such as _navbar.scss)?

### CODE REVIEW

File: dist/css/components/my-archive.css [hunk 1]
```diff
@@ -0,0 +1,17 @@
+/* Error: Undefined variable.
+ *    ,
+ * 27 | $back-ground: lighten($purple, 20%);
```
Comment (StianSto) [inline dist/css/components/my-archive.css:3]: getting an error here. $purple doesn't exist i think. typo or missing letters?

File: src/scss/_variables.scss [hunk 2]
```diff
@@ -21,7 +27,12 @@ $yellow: #ffc107;
 $green: #198754;
 $teal: #20c997;
 $cyan: #0dcaf0;
-$beige: #ccc1a4;
+
+// my color pallet
+$beige: #fff8d6;
+$turquoise: #53a6a6;
+$myYellow: #ffd500;
+$my-purple: #62366a;
```
Comment (StianSto) [inline src/scss/_variables.scss:35]: i dont mean to be a grammar nazi, but you should choose a naming convention and stick with it :) either is fine, but having both is a quick way to get errors

File: src/scss/_variables.scss [hunk 3]
```diff
@@ -71,34 +64,10 @@ $theme-colors: (
   "light": $light,
   "dark": $dark,
   "pink": $pink,
-  "purple-bg": $back-ground,
+  "my-bg": $back-ground,
+  "special": $my-purple,
 ) !default;
```
Comment (StianSto) [inline src/scss/_variables.scss:69]: remove !default tag on your own variables

File: dist/css/components/my-archive.css (hunk 1 above)
Comment (MegumiKim) [reply dist/css/components/my-archive.css:3]: Thanks for caching! This is fixed now.

File: src/scss/_variables.scss (hunk 2 above)
Comment (MegumiKim) [reply src/scss/_variables.scss:35]: You are right, changed to myPurple.

File: src/scss/_variables.scss (hunk 3 above)
Comment (MegumiKim) [reply src/scss/_variables.scss:69]: removed !default

File: dist/css/components/my-archive.css [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+ *    |                       ^^^^^^^
+ *    '
+ *   src/scss/components/my-archive.scss 27:23  root stylesheet */
+
+body::before {
```
Comment (Fermain) [inline dist/css/components/my-archive.css:8]:
```suggestion
body {
```

This style is currently targetting the :before pseudoselector, which won't affect the rest of the site. This seems like a mistake.

Comment (Fermain) [reply dist/css/components/my-archive.css:8]: Can you explain the purpose of this to me? It looks like an error message from SASS

**Review by Fermain — COMMENTED** [review]
This work shows a good understanding and use of bootstrap + sass for customisation. The only issue I have is the `:before` style which I suspect was auto generated by SASS.

File: dist/css/components/my-archive.css (hunk 4 above)
Comment (MegumiKim) [reply dist/css/components/my-archive.css:8]: Hi Oliver! Oops, this my my-archive file was just a memo for myself. This is basically a collection of stuff I thought I might use somewhere in future. I forgotten taken it out from the folder actually, definitely not for distribution. I removed the file now. Thanks for spotting.

**Pull request merged**